src/cameras/src/pm/Camera.cpp

```diff
@@ -607,7 +721,7 @@ bool pm::Camera<F>::initSpeedTable() {
 
 
 template<FrameConcept F>
-bool pm::Camera<F>::setExp(const ExpSettings& settings) {
+bool pm::Camera<F>::setupContExp() {
```

Comment:
This method will probably be updated at some point to support snap and sequence captures so I don't think we should change the name

Reply:
I think the diff is showing up weird, this is a new method I added, `setExp` still exists